Snakemake/workflow/rules/merge_samples_vcf.smk

```diff
@@ -25,18 +25,29 @@ if len(allFiles) != 1:
                 expand('/{{chromosome}}/{{chromosome}}_{file}.filtered.vcf.{ext}', file=allFiles, ext=['gz', 'gz.csi'])])
         params:
             duplicated = '{chromosome}.ids'
-       # conda: 'bcftools'
         resources: cpus=1, mem_mb=64000, time_min=60
         run:
             import os
+
+            for file in input:
+                shell("wc -l {file} >> files_len.txt")
```

Comment:
This sorts by the number of lines - which is SNPs, not samples. Is this ok?

Reply:
I guess this is about the number of individuals, correct @gmafrafortuna ? We could probably extract the number of individuals using vcf/bcftools. I also found this awk one-liner: `awk '{if ($1 == "#CHROM"){print NF-9; exit}}' input_vcf_file.vcf` This looks at the #CHROM line in the vcf file which is the line of the column header (which includes individual names).